documentation/library-reference/source/language-extensions/numbers.rst

```diff
@@ -1075,7 +1077,7 @@ example:
 
 .. code-block:: dylan
 
-    ga/+ (5, 4);
+    ga/+(5, 4);
```

Comment:
In our codebase `generic/`, `generic-`, or `big/` is used; it might make sense to stick with one of these.

Reply:
done